doc/man1/cmds.pod

```diff
@@ -0,0 +1,153 @@
+=pod
```

Comment:
Because there's *always* an output file with the same basename as the POD, it's probably better to rename it to `openssl-cmds`.  I don't see a need to add that name to the NAME section, though

Reply:
Renamed the file as requested and tested the installed manpages.